app/routers/courses.py

```diff
@@ -0,0 +1,123 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+import app.database.schemas.courses as schemas
+import app.database.models.courses as models
+from app.controllers import courses_crud as crud
+from app.database.database import get_db
+from app.utils import error_responses, enums
+
+router = APIRouter(prefix="/courses", tags=["Courses"])
+
+
+@router.get('/count/', response_model=int)
+def get_courses_count(db: Session = Depends(get_db)):
+    """
+    This function returns the number of courses in the database
+    :param db: The database session to use
+    :return: The number of courses in the database
+    """
+    return db.query(models.Course).count()
```

Comment:
It would be better if we ensure that our controller for the explicit router is doing all the model manipulation. In some functions, there is 'crud.some_method()', but in some there is direct 'db.query()'.
It is not common for controllers to be named 'something_crud', and they should be dedicated to doing all the things in routers. In complex projects, they can call the service layer which will do model manipulation.